README.md

```diff
@@ -84,12 +84,15 @@ You will need the following to be able to run the project.
 git clone https://github.com/roerohan/8086.js.git
 ```
 
-2. Install NPM packages
+2. Install NPM packages using (npm or yarn)
 
 ```sh
-npm install
+npm install or yarn install
```

Comment:
It would be nice if we had npm install and yarn install on separate lines, like:

### Using npm
```sh
npm install
```

### Using yarn
```sh
yarn install
```

Reply:
I fixed it, thanks for contributing!